### PR TITLE
Redirect to the relevant page after accepting invitation

### PR DESCRIPTION
**Pull request by Yetizone**
Closes https://github.com/opencollective/opencollective/issues/3453

# Description

These changes make the accept button callback to redirect to the collective page.
As mentioned in the issue conversation, this only happens when there's a single pending invitation.

# Screenshots
I've recorded two gifs. 
In the first gif the user accepts a single pending invitation and the redirect does happen:

single-invite

In the second gif the user accepts two pending invitations and the redirect doesn't happen:

multiple-invites

Let me know if there's anything else that needs to be done :)

### CODE REVIEW

File: components/ReplyToMemberInvitationCard.js [hunk 1]
```diff
@@ -62,6 +63,10 @@ const ReplyToMemberInvitationCard = ({ invitation, isSelected, refetchLoggedInUs
   const hasReplied = data && typeof data.replyToMemberInvitation !== 'undefined';
 
   const buildReplyToInvitation = accept => async () => {
+    if (accept && redirectOnAccept) {
+      await Router.pushRoute(`/${invitation.collective.slug}`);
+    }
```
Comment (Betree) [inline components/ReplyToMemberInvitationCard.js:68]: The redirects needs to happen after we accept the invitation (`sendReplyToInvitation`). Ideally, it should also happen after `refetchLoggedInUser` to make sure we refresh the permissions before loading the page.

**Review by Betree — CHANGES_REQUESTED** [review]
Nice implementation! We should move a bit the check for `redirectOnAccept`, but otherwise it's a good solution to https://github.com/opencollective/opencollective/issues/3453

File: components/ReplyToMemberInvitationCard.js (hunk 1 above)
Comment (Yetizone) [reply components/ReplyToMemberInvitationCard.js]: Done!! Should I also change the commit message to match the new PR description ?

**Comment by Yetizone** [comment]
This is a rewrite of a previous comment that was hidden so I'm putting it here for visibility: 

I noticed the PR title was changed, should I also change the commit message to mirror it ?

**Comment by Betree** [comment]
> I noticed the PR title was changed, should I also change the commit message to mirror it ?

No it's fine, your commit message is great actually!

**Review by Betree — APPROVED** [review]
This is working great, good job @Yetizone!

**Comment by Betree** [comment]
Moving the PR to https://github.com/opencollective/opencollective-frontend/pull/5709 as we need ot sync it with API to update the E2E tests. Btw @Yetizone you're now a member of https://github.com/orgs/opencollective/teams/contributors/members so you don't need to use a fork anymore, you can commit directly to the main repo :slightly_smiling_face:

**Pull request closed without merging**